fecalc.js

```diff
@@ -0,0 +1,292 @@
+// FE:H Custom Calc
+// LF<Trap Card - Kunai w/ Chain
+var math = require('mathjs');
+var _und = require('underscore');
+
+module.exports = {
+    init() {
+        return init();
+    },
+    feCustomCalc( characters, callback ) {
+        return feCustomCalc( characters, callback );
+    },
+    feParse( data ) {
+        return feParse( data );
+    },
+    feCreateChar( parseddata ) {
+        return feCreateChar( parseddata );
+    },
+    feResults (characters, callback) {
+        return feResults( characters, callback );
+    }
+}
+
+// init function
+function init() {
+}
+
+//feCustomCalc takes data and throws back an endmessage 
+// set adv 
+// number vs number
+// additive values before or after numbers
+// hp, atk, spd, def, res, [(d)braveweapon, (e)ffective, ra(n)ged, (m)agic,
+//     c(o)unterable, (r)ed, (b)lue, (g)reen, (c)olorless, (a)dvantage]
+// Damage: ceil(floor(floor(Atk * ADV) * EFF) - DEF/RES)
+// adv array [ initiator, defender ] 
+function feCustomCalc( characters, callback ) {
+    var dmg = [];
+    var adv = [1, 1], eff = [1, 1];
+    // Set ADV / DADV 
+    if ( characters[0].flags.includes("r") ) {
+        if ( characters[1].flags.includes("b") ) {
+            if ( characters[0].flags.includes("a") || characters[1].flags.includes("a") ) {
+                adv = [0.6, 1.4]; 
```

Comment:
I'd recommend making Constants for the Adv/DisAdv numbers. If we ever need to make an edit to these values we can change it in one place. Make a constant for the Characters as well.